src/llvm/ext/llvm_ext.cc

```diff
@@ -87,7 +87,7 @@ LLVMMetadataRef LLVMDIBuilderCreateCompileUnit2(DIBuilderRef Dref, unsigned Lang
 #endif
 }
 
-LLVMMetadataRef LLVMDIBuilderCreateFunction(
+LLVMMetadataRef LLVMDIBuilderCreateFunction2(
```

Comment:
Adding a `2` may still conflict with LLVM functions. Maybe the functions should be prefixed `LLVMExt` or, better, match the now official functions signature (but that's more work).

Reply:
Matching the official one is not possible since the `llvm_ext.cc` may act as a common denominator for the API across llvm versions.